gql/builder.go

```diff
@@ -390,7 +394,7 @@ func (ob *ObjectBuilder) resolveObjectByName(p graphql.ResolveTypeParams) *graph
 //    }
 //  }
 //
-func ResolveListField(name string, parent string) graphql.FieldResolveFn {
+func ResolveListField(name string, parent string, nonNull bool) graphql.FieldResolveFn {
```

Comment:
this is an exported func, so this change may break others. You might want to wrap this to enable others to continue to use the original implementation